packages/core/content-releases/server/src/services/release.ts

```diff
@@ -88,30 +88,47 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
       hasEntryAttached: false,
     }
   ) {
-    const whereActions = hasEntryAttached
-      ? {
-          // Find all Releases where the content type entry is present
+    const getWhereObject = async (hasEntryAttached: boolean = false) => {
```

Comment:
I don't think I like a function defined in this method.

I have two ideas:

1. We don't get the where clause conditionally but instead do all the logic and return separate entityService.findMany's like:
```ts
if (hasEntryAttached) {
		// whatever logic needed for this case
		return entityService.findMany(blabla, {
			....filters logic etc for hasEntryAttached === true
		})
}

// whatever logic needed for this case
return entityService.findMany(blabla, {
	....filters logic etc for hasEntryAttached === false
})
```

2. Create separate methods instead of passing the hasEntryAttached param:
- findManyWithContentTypeEntryAttached
- findManyWithoutContentTypeEntryAttached

Reply:
I definitely liked the 2nd solution more!